app/controllers/admin/inventory_items_controller.rb

```diff
@@ -14,6 +16,10 @@ class InventoryItemsController < ResourceController
 
     private
 
+    def permit_params
+      params.require(:inventory_item).permit!
```

Comment:
I don't think we should be using `#permit!`...

Reply:
done in new commit :+1: